python/main.py

```diff
@@ -52,6 +54,17 @@ async def lifespan(app: FastAPI):
 )
 
 
+def save_image(file: UploadFile) -> str:
+    content = file.file.read()  # ← 画像のデータを全部読み込む（バイナリデータ(jpgとかtextじゃないデータ)）???
+    hashed_name = hashlib.sha256(content).hexdigest()  # ハッシュ化??? hashed-valueが返るの？
+    file_name = f"{hashed_name}.jpg"
+    file_path = os.path.join("images", file_name)
```

Comment:
もしかしたらL46のpathを使ってあげるといいかもしれません。

```suggestion
    file_path = f"{images}/{file_name}"
```